pkg/registry/pod/strategy.go

```diff
@@ -253,6 +255,21 @@ func LogLocation(getter ResourceGetter, connInfo client.ConnectionInfoGetter, ct
 	if opts.Previous {
 		params.Add("previous", "true")
 	}
+	if opts.Timestamps {
```

Comment:
Instead of doing this, can't you just use https://github.com/kubernetes/kubernetes/blob/1a88be1aef8a9ed1219ceb9079330986ecafd46d/pkg/conversion/queryparams/convert.go#L99 ?